pkgs/development/python-modules/bqplot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, jupyter-packaging
+, bqscales
+, ipywidgets
+, numpy
+, pandas
+, traitlets
+, traittypes
+}:
+
+buildPythonPackage rec {
+  pname = "bqplot";
+  version = "0.12.36";
+
+  format = "pyproject";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-alU+Vf28xPYuBWeb10slSdHvvkPhIK07C8sDplx5Ia8=";
+  };
+
+  nativeBuildInputs = [
+    jupyter-packaging
+  ];
+
+  propagatedBuildInputs = [
+    bqscales
+    ipywidgets
+    numpy
+    pandas
+    traitlets
+    traittypes
+  ];
+
+  # no tests in PyPI dist
```

Comment:
Can we use `fetchFromGitHub`?

Reply:
Of course I would prefer this, but as usual with these ipy*/jupyter packages it's not immediately obvious how to get the javascript files to be built/included in the right way.

Reply:
I added a comment about this.